modoboa/admin/management/commands/subcommands/_mx.py

```diff
@@ -79,6 +79,11 @@ def query_dnsbl(self, mx_list, provider):
             pattern = "{}.{}.".format(reverse, provider)
             try:
                 result = socket.gethostbyname(pattern)
+                #result from dnsbl is in ipv4 format
+                splited_result = result.split(".")
+                if int(splited_result[-1]) <= 15:
+                    #Typical dnsbl result : 127.255.255.[1-15] (depends on services)
+                    result = False
```

Comment:
That's indeed better like this but if it depends on services then I guess we should store some kind of config somewhere. Do you know if it's documented somewhere?

Reply:
It really depends on the provider (spamhaus: https://www.spamhaus.org/faq/section/DNSBL%20Usage#200, spamcop: https://www.spamcop.net/fom-serve/cache/291.html, etc.). When ip is flagged result is 127.0.0.[1-15] and this is share among all providers I checked.